code/game/turfs/simulated/minerals.dm

```diff
@@ -185,6 +185,12 @@
 		/turf/closed/mineral/silver = 6, /turf/closed/mineral/plasma = 15, /turf/closed/mineral/iron = 40,
 		/turf/closed/mineral/gibtonite = 2, /turf/closed/mineral/bscrystal = 1)
 
+/turf/closed/mineral/random/low_chance/volcanic
```

Comment:
This ought to be in `face/code/game/turfs/simulated/minerals.dm` just to avoid merge conflicts in the future

Reply:
I'll make another commit for that then